src/pages/layer-2.tsx

```diff
@@ -40,153 +53,114 @@ import { getImage } from "../utils/image"
+    isLightGrayBg?: boolean
   }
-`
+) => (
+  <Box
+    px={8}
+    py={12}
+    width="full"
+    {...(props.isLightGrayBg && { background: "layer2ContentSecondary" })}
+  >
+    {props.children}
+  </Box>
+)
 
-const RollupCard = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  background: ${(props) => props.theme.colors.ednBackground};
-  border-radius: 2px;
-  border: 1px solid ${(props) => props.theme.colors.lightBorder};
-  padding: 1.5rem;
-  flex: 50%;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex: 100%;
-  }
-`
+const StyledInfoIcon = () => (
+  <Icon
+    as={MdInfoOutline}
+    color="text"
+    mr={2}
+    opacity={0.8}
+    boxSize="full"
+    _hover={{ color: "primary" }}
+    _active={{ color: "primary" }}
+    _focus={{ color: "primary" }}
+  />
+)
 
-const StatsContainer = styled.div`
-  display: flex;
-  justify-content: center;
-  align-content: center;
-  margin-bottom: 4rem;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex-direction: column;
-  }
-`
+const TwoColumnContent = (props: ChildOnlyType) => (
+  <Flex
+    justifyContent="space-between"
+    gap={16}
+    flexDirection={{ base: "column", lg: "row" }}
+    alignItems={{ base: "flex-start", lg: "normal" }}
+    {...props}
+  />
+)
 
-const StatBox = styled.div`
-  display: flex;
-  flex-direction: column;
-  flex: 33%;
-  padding: 0 20px;
-  text-align: center;
-  align-content: center;
-  justify-content: center;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex: 100%;
-  }
-`
+const StatDivider = () => {
+  const responsiveOrientation = useBreakpointValue<DividerProps["orientation"]>(
+    { base: "horizontal", md: "vertical" }
+  )
+  return (
+    <Divider
+      orientation={responsiveOrientation}
+      borderColor="homeDivider"
+      my={{ base: 8, md: 0 }}
+    />
+  )
+}
 
-const StatPrimary = styled.p`
-  font-weight: bold;
-  font-size: ${(props) => props.theme.fontSizes.xl};
-  color: ${(props) => props.theme.colors.primary};
-  font-family: monospace;
-`
+const StatBox = (props: ChildOnlyType) => (
+  <Center
+    flexDirection="column"
+    flex={{ base: "100%", md: "33%" }}
+    textAlign="center"
+    px={5}
+    {...props}
+  />
+)
 
-const StatSpan = styled.div`
-  display: flex;
-  justify-content: center;
-  gap: 0.5rem;
-`
+const StatPrimary = (props: { content: string }) => (
+  <Text
+    color="primary"
+    fontFamily="monospace"
+    fontWeight="bold"
+    fontSize="2rem"
+  >
+    {props.content}
+  </Text>
+)
 
-const StatDescription = styled.p`
-  opacity: 0.8;
-  margin: 0;
-`
+const StatSpan = (props: ChildOnlyType) => (
+  <Flex justifyContent="center" gap={2} {...props} />
+)
+
+const StatDescription = (props: ChildOnlyType) => (
+  <Text opacity={0.8} m={0} {...props} />
+)
+
+const Layer2CardGrid = (props: ChildOnlyType) => (
+  <SimpleGrid columns={4} gap={8} minChildWidth="280px" {...props} />
```

Comment:
It seems that by using `minChildWidth` it ignores the `columns` and uses `auto-fit`. <https://chakra-ui.com/docs/components/simple-grid#auto-responsive-grid>

![image](https://user-images.githubusercontent.com/468158/223476529-8771bf72-ed60-4662-b9a0-629c26bcc12d.png)

I think we should need to use the `Grid` component in this case.

Reply:
@pettinarip oh shoot! 🤦🏼 

Are you suggesting to go use `Grid` because of the `gridTemplateColumns` prop?

I can go back to using the original value for `gridTemplateColumns` and still use `SimpleGrid` via `templateColumns`

```tsx
const Layer2CardGrid = (props: ChildOnlyType) => (
  <SimpleGrid
    // This value below is in prod
    templateColumns="repeat(auto-fill, minmax(min(100%, 280px), 1fr))"
    gap={8}
    {...props}
  />
)
```

Reply:
Oh no, go with your approach if that solves the problem. I was just giving an idea. Didn't know that you could use `templateColumns` on the `SimpleGrid`.

Reply:
However, do you feel that `auto-fill` should be used here? seems that we need the exact number of columns here. That would be `4` => `templateColumns="repeat(4, minmax(min(100%, 280px), 1fr))"`

Reply:
Using `4` instead of `auto-fill` is good here. If the container was any wider at the desktop view, it would look odd to me having more than 4 in a row.

Reply:
We needed the `auto-fill` in this case, it was not wrapping correctly.

![image](https://user-images.githubusercontent.com/468158/227235402-9de53df6-006c-4583-b336-2e6895cc442e.png)

Sorry for all the back and forth. Now it looks nice.